### DISCUSSION TITLE
How can I find the example markdown

### DISCUSSION THREAD

**Discussion by BearGrass** [cotes2020/jekyll-theme-chirpy#1877] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

The writing examples on this website are what I need, but I can't view the original text. Where can I view it?

url: https://chirpy.cotes.page/posts/text-and-typography/

### Operations you have already tried

no

**Comment by huanyushi** [comment]
See [here](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/_posts/2019-08-08-text-and-typography.md?plain=1).

**Comment by kungfux** [comment, marked as answer]
While the [Text and Typography](https://chirpy.cotes.page/posts/text-and-typography/) shows how the result will look like, [Write a new post](https://chirpy.cotes.page/posts/write-a-new-post/) shows how to do it. And you can see the [source code of that page](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/_posts/2019-08-08-text-and-typography.md?plain=1) too.

**Reply by BearGrass** [reply]
This is exactly what I wanted, thank you
